packages/client/src/store/modules/users.js

```diff
@@ -61,6 +61,11 @@ export default {
       await fetchApi.post('/api/organizations/:organizationId/users', user);
       await dispatch('fetchUsers');
     },
+    async updateUser({ commit }, user) {
+      await fetchApi.put(`/api/organizations/:organizationId/users/${user.id}`, user);
+      const data = await fetchApi.get('api/sessions');
```

Comment:
I'm not sure about this... I am calling api/sessions since the user object the put is returning at the moment is incomplete. But I can resolve that within the put route by calling the same getUsers function /sessions uses so there's only one network request?